contrib/format-maprdb/src/main/java/org/apache/drill/exec/store/mapr/db/json/MaprDBJsonRecordReader.java

```diff
@@ -530,8 +530,10 @@ public void close() {
 
   @Override
   public String toString() {
-    StringBuilder sb = new StringBuilder("MaprDBJsonRecordReader[Table=")
-        .append(table.getPath());
+    StringBuilder sb = new StringBuilder("MaprDBJsonRecordReader[Table=");
+    if (table != null) {
```

Comment:
For the case when `table` is null, the string will be formed incorrectly - `Table=` should be appended only if `table != null`

Reply:
Thanks for pointing it out, done.